packages/suite/src/views/settings/coins/FirmwareTypeSuggestion.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import * as suiteActions from '@suite-actions/suiteActions';
+import * as routerActions from '@suite-actions/routerActions';
+import { Translation } from '@suite-components';
+import { TextColumn } from '@suite-components/Settings';
+import { SettingsAnchor } from '@suite-constants/anchors';
+import { useActions, useDevice } from '@suite-hooks';
+import { isBitcoinOnly } from '@suite-utils/device';
+import { Button, Card } from '@trezor/components';
+
+const StyledCard = styled(Card)`
+    align-items: flex-start;
+    border-left: 10px solid ${props => props.theme.TYPE_GREEN};
+    box-shadow: 0 2px 5px 0 ${props => props.theme.BOX_SHADOW_BLACK_20};
+    flex-direction: row;
+    justify-content: space-between;
+`;
+
+const StyledButton = styled(Button)`
+    display: inline;
+`;
+
+export const FirmwareTypeSuggestion = () => {
+    const { goto, setFlag } = useActions({
+        goto: routerActions.goto,
+        setFlag: suiteActions.setFlag,
+    });
+    const { device } = useDevice();
+
+    const bitcoinOnlyFirmware = device && isBitcoinOnly(device);
+
+    const handleClose = () => setFlag('firmwareTypeBannerClosed', true);
+
+    return (
+        <StyledCard>
+            <TextColumn
+                description={
+                    <Translation
+                        id={
+                            bitcoinOnlyFirmware
+                                ? 'TR_SETTINGS_COINS_UNIVERSAL_FIRMWARE_SUGGESTION'
+                                : 'TR_SETTINGS_COINS_BITCOIN_FIRMWARE_SUGGESTION'
+                        }
+                        values={{
```

Comment:
This could also be placed in the body of `FirmwareTypeSuggestion` to avoid having such a stretched JSX output

Reply:
0a51f2bc16851cae212180913925531c861c696b